src/control-plane/backend/lambda/api/service/stack.ts

```diff
@@ -211,7 +211,7 @@ export class StackManager {
     for (let s of stackStatusDetails) {
       if (s.stackStatus === undefined) {
         miss = true;
-      } else if (s.stackStatus.endsWith('_FAILED')) {
+      } else if (s.stackStatus.endsWith('_FAILED') || s.stackStatus.endsWith('_ROLLBACK_COMPLETE')) {
```

Comment:
I'm not sure it's related to #281. The status of pipeline is **ACTIVE** when updating the existing configuration.

Reply:
sorry, confused with #279